packages/server-wallet/src/wallet/index.ts

```diff
@@ -160,8 +161,8 @@ export class Wallet implements WalletInterface {
   }
 
   async pushMessage(message: AddressedMessage): MultipleChannelResult {
-    const channelIds = await Channel.transaction(async tx => {
-      return await Store.pushMessage(message, tx);
+    const channelIds = await Channel.transaction(async () => {
+      return await Store.pushMessage(message);
     });
```

Comment:
The transaction here is not meaningful, since `Store.pushMessage` would execute in separate transactions.

This might be confusing coming from the browser wallet, but Dexie uses some magical promise-local data to keep track of _the_ current transaction within the execution of some async code.

Objection doesn't do this, so if you want to execute some queries within the context of a transaction, you need a handle on that transaction. IE. you are allowed to execute a concurrent transaction, if you wish. You can see an example of that [here](https://github.com/statechannels/statechannels/blob/master/packages/server-wallet/src/models/__test__/channel.test.ts#L32-L43).